lib/fathead/perl_doc/Parsers/parse.pl

```diff
@@ -155,6 +155,7 @@ sub doc_fullurl {
 # Parsers for other keys (basenames) will only run on the matching file.
 my %parser_map = (
     'index-faq'       => ['parse_faq'],
+    'index-language'  => ['parse_faq'],
```

Comment:
@jbarrett Well, it worked before - it might just be they are parsing the same header levels and content, but it's probably best to keep an eye on it for the future.